src/main.c

```diff
@@ -320,9 +320,12 @@ static void add_imported_symbols(struct CompileState *compst)
         for (AstToplevelNode *ast = to->ast; ast->kind == AST_TOPLEVEL_IMPORT; ast++) {
             AstImport *imp = &ast->data.import;
             struct FileState *from = find_file(compst, imp->path);
-            assert(from);
+            assert(from);
 
-            for (struct ExportSymbol *es = from->pending_exports; es->name[0]; es++) {
+            for (struct ExportSymbol *es = from->pending_exports; es->name[0]; es++) {
+                if (es->name == '*') {
+                    printf("WOW, it successed!                                             sd");
+                }
                 if (!strcmp(imp->symbolname, es->name)) {
```

Comment:
I think this is the line that should be changed. If we are importing a symbol named `*`, it means we should match all exported symbols.

```suggestion
                if (!strcmp(imp->symbolname, es->name) || !strcmp(imp->symbolname, "*")) {
```